scripts/shared/deploy.sh

```diff
@@ -123,6 +142,7 @@ declare_kubeconfig
 
 load_library deploy DEPLOYTOOL
 deploytool_prereqs
+[[ "$PROVIDER" == kind ]] || run_all_clusters schedule_dummy_pod
```

Comment:
Isn’t this the wrong way round? If `schedule_dummy_pod` is supposed to run with kind,

```
[[ "$PROVIDER" != kind ]] || run_all_clusters schedule_dummy_pod
```